snpe/tutorials/Inference_notebook_proposal.ipynb

```diff
@@ -0,0 +1,691 @@
+       "                var nbb_cells = Jupyter.notebook.get_cells();\n",
+       "                for (var i = 0; i < nbb_cells.length; ++i) {\n",
+       "                    if (nbb_cells[i].input_prompt_number == nbb_cell_id) {\n",
+       "                        if (nbb_cells[i].get_text() == nbb_unformatted_code) {\n",
+       "                             nbb_cells[i].set_text(nbb_formatted_code);\n",
+       "                        }\n",
+       "                        break;\n",
+       "                    }\n",
+       "                }\n",
+       "            }, 500);\n",
+       "            "
+      ],
+      "text/plain": [
+       "<IPython.core.display.Javascript object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "def infer_and_save_posterior(\n",
+    "    device: str, simulator_type: str, simulation_type: str, params: Dict\n",
+    ") -> None:\n",
+    "    parameter_prior = sbi_utils.BoxUniform(\n",
+    "        low=torch.tensor([0.0, 0.0, 0.0, 0.0, 0.5, 0.25, 0.25, 0.5]).type(\n",
+    "            torch.FloatTensor\n",
+    "        ),\n",
+    "        high=torch.tensor([4.0, 4.0, 1.0, 1.0, 1.0, 0.75, 0.75, 1.0]).type(\n",
+    "            torch.FloatTensor\n",
+    "        ),\n",
+    "        device=device,\n",
+    "    )\n",
+    "    inferrer = inference_class.TimeSeriesInference(\n",
+    "        parameter_prior=parameter_prior, device=device\n",
+    "    )\n",
+    "    inferrer.load_simulator(\n",
+    "        dirname=ARTIFACT_PATH,\n",
+    "        simulator_type=simulator_type,\n",
+    "        simulation_type=simulation_type,\n",
+    "    )\n",
+    "    batch_size = params.pop(\"batch_size\")\n",
+    "    learning_rate = params.pop(\"learning_rate\")\n",
+    "    hidden_features = params.pop(\"hidden_features\")\n",
+    "    num_transforms = params.pop(\"num_transforms\")\n",
+    "    inferrer.infer_snpe_posterior(\n",
+    "        embedding_net_conf=params,\n",
+    "        batch_size=batch_size,\n",
+    "        learning_rate=learning_rate,\n",
+    "        hidden_features=hidden_features,\n",
+    "        num_transforms=num_transforms,\n",
+    "    )\n",
+    "    inferrer.save_inference(ARTIFACT_PATH)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88521c7b",
+   "metadata": {},
+   "source": [
+    "Defining the dictionary of parameters that has to be provided as argument to `infer_and_save_posterior`"
```

Comment:
These parameters are all found by going through a process of Bayesian hyperparameter optimization (in `timeseries_cnn_tuning.py`). So we should say here that there was this hyperparameter optim. process and that these parameters didn't just emerge randomly.

Reply:
I've added a mention to this on the text